prisma/schema.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE "User" (
+    "id" TEXT NOT NULL,
+    "name" TEXT,
+    "email" TEXT,
+    "emailVerified" TIMESTAMP(3),
+    "image" TEXT,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
```

Comment:
Should we allow `updatedAt` to be null? If we want it to be NOT NULL, perhaps we should default it to `CURRENT_TIMESTAMP` too.

Reply:
sounds good - done.